pixz/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Maxim Moseychuk ("fshp") <franchesko.salias.hudro.pedros@gmail.com>
+
+pkgname=pixz
+pkgver=1.0.7
+pkgrel=2
+pkgdesc="Parallel, indexed xz compressor"
+arch=('i686' 'x86_64')
+url="https://github.com/vasi/pixz"
+license=('custom')
+depends=('libarchive')
+makedepends=('asciidoc' 'libarchive-devel')
+source=(https://github.com/vasi/pixz/releases/download/v$pkgver/${pkgname}-$pkgver.tar.gz)
```

Comment:
consider deriving this link from the url variable you provided, in case this URL changes to another gh repo, it would be easier to update just one line